ocs_ci/deployment/deployment.py

```diff
@@ -876,6 +880,30 @@ def setup_local_storage(storageclass):
         f"{constants.OPERATOR_NODE_LABEL}"
     )
 
+    ocp_version = get_ocp_version()
+    ocp_ga_version = get_ocp_ga_version(ocp_version)
+    if not ocp_ga_version:
+        local_storage_namespace = constants.NON_GA_LOCAL_STORAGE_NAMESPACE
+        optional_operators_data = templating.load_yaml(
+            constants.LOCAL_STORAGE_OPTIONAL_OPERATORS, multi_document=True
+        )
+        logger.info(
+            "Creating temp yaml file with optional operators data:\n %s",
+            optional_operators_data
+        )
+        optional_operators_yaml = tempfile.NamedTemporaryFile(
+            mode='w+', prefix='optional_operators', delete=False
+        )
+        templating.dump_data_to_temp_yaml(
+            optional_operators_data, optional_operators_yaml.name
+        )
+        with open(optional_operators_yaml.name, 'r') as f:
+            logger.info(f.read())
+        logger.info("Creating optional operators CatalogSource and ICSP")
```

Comment:
what is ICSP?

Reply:
icsp stand for imagecontentsourcepolicy

Reply:
ImageContentSourcePolicy

Reply:
I changed it to `ImageContentSourcePolicy` so there is no confusion